BUILDING.md

```diff
@@ -8,6 +8,29 @@ You can skip to the [setup guide for Kubernetes](QUICKSTART-EKS.md) or the [setu
 
 ### Dependencies
 
+#### System Requirements
+
+The build process artifacts and resulting images can consume in excess of 4GB in the local directory.
```

Comment:
Let's bump up this number quite a bit as @bcressey mentioned: we're probably safe to say around 80GB.